oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthenticationCallbackHandler.java

```diff
@@ -486,11 +484,15 @@ private Map<String, String> buildDimensions(
             LOG.info(
                     "No mfa method to set. User is either authenticated or signing in from a low level service");
         }
-        VectorOfTrust vtr = clientSession.getVtrWithLowestCredentialTrustLevel();
-        var mfaRequired = !vtr.getCredentialTrustLevel().equals(CredentialTrustLevel.LOW_LEVEL);
+        var orderedVtrList = VectorOfTrust.orderVtrList(clientSession.getVtrList());
+        var mfaRequired =
+                !orderedVtrList
+                        .get(0)
+                        .getCredentialTrustLevel()
+                        .equals(CredentialTrustLevel.LOW_LEVEL);
         var levelOfConfidence = LevelOfConfidence.NONE.getValue();
-        if (vtr.containsLevelOfConfidence()) {
-            levelOfConfidence = vtr.getLevelOfConfidence().getValue();
+        if (orderedVtrList.get(0).containsLevelOfConfidence()) {
```

Comment:
I think we said we were going to write a comment here about the assumption we've made here implicitly, and why it's justified?

Reply:
That comment is in the identity helper. Here we want the `RequestedLevelOfConfidence` which is the lowest requested value (hence ordering and taking the first)